Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/Update.ts

```diff
@@ -59,7 +59,7 @@ export async function updateDocumentById(
         Logger.debug(`${moduleName}.updateDocument: Inserting document id ${id} failed due to invalid references`, traceId);
         updateResult = {
           response: 'UPDATE_FAILURE_REFERENCE',
-          failureMessage: `Reference validation failed: ${failures.join(',')}`,
+          failureMessage: { error: { message: 'Reference validation failed', failures } },
```

Comment:
@bradbanister @andonyns the weird indexing issue we explored this morning was due to a previous iteration of the line above, where I dereferenced like this:

```failureMessage: { error: { message: 'Reference validation failed', ...failures } },```